### PR TITLE
Upload file docs

### PR DESCRIPTION
**Pull request by jandonyada**
### Which issues does this fix?
https://github.com/workato/workato/pull/21902

### PR submission checklist
- [x] Optimised every new image
- [x] Checked all content for grammatical errors (using Grammarly, for example)
- [x] Added new files to SUMMARY.md

### CODE REVIEW

File: workbot/workbot-actions.md [hunk 1]
```diff
@@ -233,3 +234,65 @@ When returning menu options, you can group menu options together by setting **Gr
...
+        Upload a file to a specified Slack channel or direct message (DM).
+        </td>
+      <td>
+        Use the <kbd>Channel/DM</kbd> datapill from a Post command trigger, or key in the channel name e.g. <b>#general</b> or username e.g. <b>@johndonahue</b>.<img src="/assets/images/workbot/workbot-actions/channel-dm.png"></img>
+      </td>
+    </tr>
+    <tr>
+      <td>Initial comment</td>
+      <td>Simple message to post together with the uploaded file.</td>
+      <td>Include a simple message to give more context to your file.<img src="/assets/images/workbot/workbot-actions/upload-file-with-message.png"></img></td>
```
Comment (eeshansim) [inline workbot/workbot-actions.md]: I don't see the message in this screenshot

File: workbot/workbot-actions.md [hunk 2]
```diff
@@ -233,3 +234,65 @@ When returning menu options, you can group menu options together by setting **Gr
...
+    <tr>
+      <td>File type</td>
+      <td>Type of file</td>
+      <td>Provide the file type of your file. For the full list of file types which Slack supports, see <a href='https://api.slack.com/types/file#file_types'>here.</td>
+    </tr>
+    <tr>
+      <td>File content</td>
+      <td>Contents of the file</td>
+      <td>
+        Use a <kbd>Content</kbd> datapill from the output of preceding action or trigger, e.g. a File contents datapills from a Google Drive Download attachment action.
```
Comment (eeshansim) [inline workbot/workbot-actions.md]: An image of this input field with the datapill mapped from Drive output tree will be better. Make sure to highlight the link between mapped datapill and the datatree by clicking on the pill before taking the screenshot.

File: workbot/workbot-actions.md (hunk 1 above)
Comment (jandonyada) [reply workbot/workbot-actions.md]: Strange. Anw, updated.

File: workbot/workbot-actions.md (hunk 2 above)
Comment (jandonyada) [reply workbot/workbot-actions.md]: Updated

**Pull request merged**